rdrf/rdrf/templates/rdrf_cdes/contexts.html

```diff
@@ -153,22 +158,34 @@
 
 {% block content %}
 
-    <p>
-        <select id="registry_options">
-            {% for reg in registries %}
-                <option id="{{ reg.code }}">{{ reg.name }}</option>
-            {% endfor %}
-        </select>
-        <span id="loading_text"><i>Loading...</i></span>
-    </p>
-
-    <table id="context_list_table" class="table table-striped table-hover">
-        <thead>
-            <tr>
-                {% for column in columns %}
-                    <th>{{ column.label|safe }}</th>
+    {% if not registries %}
+        <div class="alert alert-danger">
```

Comment:
seems ok, if there are no registries , what happens to the JS that hooks up  the ( non existent) select with id registry_options?